plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2GroupFacet.java

```diff
@@ -84,12 +88,28 @@ protected void doInit(final Configuration configuration) throws Exception {
     this.mavenFacet = facet(MavenFacet.class);
   }
 
+  @Override
+  protected void doConfigure(final Configuration configuration) throws Exception {
+    super.doConfigure(configuration);
+    if (cacheController == null) {
+      cacheController = new CacheController(1440, null);
+    }
+    cacheController.invalidateCache();
```

Comment:
in `ProxyFacetSupport` the cache controller field is always overwritten in the `doConfigure` method:

https://github.com/sonatype/nexus-oss/pull/1573/files#diff-dd100f40138a166047ddf8c45825e58dR102

and it also doesn't call `invalidateCache` so I'm curious why the group facet uses a different approach.